R/R/prophet.R

```diff
@@ -1274,9 +1288,134 @@ fit.prophet <- function(m, df, ...) {
 #' forecast <- predict(m, future)
 #' plot(m, forecast)
 #' }
-#'
 #' @export
 predict.prophet <- function(object, df = NULL, ...) {
+  
+  if (is.null(df)) {
+    m <- object
+    history.and.future <- m$history
```

Comment:
I think the variable name `history.and.future` isn't quite appropriate because while by default we always generate future dataframes that include the history, it doesn't have to. `df` may contain only future dates.